impl_core/src/test/java/io/opencensus/implcore/stats/RecordUtilsTest.java

```diff
@@ -83,6 +86,19 @@ public void testGetTagValues() {
         .inOrder();
   }
 
+  @Test
+  public void testGetTagValues_MapDeprecatedRpcTag() {
+    List<TagKey> columns = Arrays.asList(RecordUtils.RPC_STATUS, RecordUtils.RPC_METHOD);
+    Map<TagKey, TagValueWithMetadata> tags =
+        ImmutableMap.of(
+            RecordUtils.GRPC_CLIENT_METHOD, METHOD_V_WITH_MD,
+            RecordUtils.GRPC_CLIENT_STATUS, STATUS_V_WITH_MD);
```

Comment:
What happens when we have both a client method and a server method?

Reply:
Depending on what tag keys we have in the view:
1. If we have the new "client_method" or "server_method" key (or even both), everything works fine;
2. If we have the old "method" key, a random value will be picked. While without this mapping the value will always be null.

IMO this should be less an issue because eventually users should migrate to use the new keys. WDYT?

Reply:
By looking at `getTagValueForDeprecatedRpcTag()`, it seems the underlying data (columns?) is still in the old format, thus `getTagValueForDeprecatedRpcTag()` tries to get the values from the new tags in case the old tags are not set. If both "client_method" or "server_method" are set, "client_method" will be used. Is it going to be the case even after the old tags are removed?

Reply:
So, iiuc, if we have a view with and old key (e.g. "rpc_method") and tags 

1. contains tag rpc_method:v1 then v1 is used
2. otherwise, contains tag grpc_client_method:v2 then v2 is used
3. otherwise contains tag grpc_server_method:v3 then v3 is used
4. otherwise UNKNOWN_TAG_VALUE is used

Is that correct? If so, it may be worth enumerating in the unit tests. Also, when we have both grpc_client_method and grpc_server_method, which one matches closer to the original notion of rpc_method?

Reply:
> If both "client_method" or "server_method" are set, "client_method" will be used.

Apologies for the confusion, what Kun said is correct. If both are present and a view is using the old column, client method value will be picked.

> if we have a view with and old key (e.g. "rpc_method") and tags

Yes the order is correct. I added a few tests on this.

Reply:
> If both "client_method" or "server_method" are set, "client_method" will be used. Is it going to be the case even after the old tags are removed?

No, the new "client_method" and "server_method" tags will be matched properly regardless of the old tag. This change is only trying to get the values from the new tags for old tag if it's still in use. Added one more test in https://github.com/census-instrumentation/opencensus-java/pull/1854/commits/cec1f5ac0c74309f7bfc14907b610b9a560a4247 to verify.